Lib/fontbakery/profiles/outline.py

```diff
@@ -0,0 +1,290 @@
+        baseline point. As well as the baseline, the test also checks for
+        points near the x-height (but only for lower case Latin letters),
+        cap-height, ascender and descender Y coordinates.
+
+        Not all such misaligned curve points are a mistake, and sometimes the
+        design may call for points in locations near the boundaries. As this
+        test is liable to generate significant numbers of false positives, the
+        test will pass if there are more than {FALSE_POSITIVE_CUTOFF} reported
+        misalignments.
+""",
+    conditions=["outlines_dict"],
+)
+def com_google_fonts_check_outline_alignment_miss(ttFont, outlines_dict):
+    """Are there any misaligned on-curve points?"""
+    alignments = {
+        "baseline": 0,
+        "x-height": ttFont["OS/2"].sxHeight,
+        "cap-height": ttFont["OS/2"].sCapHeight,
+        "ascender": ttFont["OS/2"].sTypoAscender,
+        "descender": ttFont["OS/2"].sTypoDescender,
+    }
+    warnings = []
+    for glyphname, outlines in outlines_dict.items():
+        for p in outlines:
+            for node in p.asNodelist():
+                if node.type == "offcurve":
+                    continue
+                for line, yExpected in alignments.items():
+                    # skip x-height check for caps
+                    if line == "x-height" and (
+                        len(glyphname) > 1 or glyphname[0].isupper()
+                    ):
+                        continue
+                    if close_but_not_on(yExpected, node.y, ALIGNMENT_MISS_EPSILON):
+                        warnings.append(
+                            f"{glyphname}: X={node.x},Y={node.y} (should be at {line} {yExpected}?)"
+                        )
+        if len(warnings) > FALSE_POSITIVE_CUTOFF:
+            # Let's not waste time.
+            yield PASS, (
+                "So many Y-coordinates of points were close to boundaries that this was probably by design."
+            )
+            return
+
+    if warnings:
+        formatted_list = "\t* " + pretty_print_list(warnings, sep="\n\t* ")
+        yield WARN, Message(
+            "found-misalignments",
+            f"The following glyphs have on-curve points which"
+            f" have potentially incorrect y coordinates:\n"
+            f"{formatted_list}",
+        )
+    else:
+        yield PASS, ("Y-coordinates of points fell on appropriate boundaries.")
+
+
+@check(
+    id="com.google.fonts/check/outline_short_segments",
+    rationale=f"""
+    This test looks for outline segments which seem particularly short (less than
+    {SHORT_PATH_EPSILON}%% of the overall path length).
+
+    This test is not run for variable fonts, as they may legitimately have short
+    segments. As this test is liable to generate significant numbers of false
+    positives, the test will pass if there are more than {FALSE_POSITIVE_CUTOFF}
+    reported short segments.
+""",
```

Comment:
Rationale text must use 8-spaces of indentation and must not be manually line-wrapped. FontBakery reporters will take care of performing text layout optimized for each type of output generated (HTML, Markdown, text-terminal, etc)

Reply:
The test tester should test for this!

Reply:
check tester!

Yes, indeed, the minimum indentation level could be auto detected with a code-test. But only a human can grasp the difference between line breaking to separate paragraphs versus line-braking in the middle of sentences to manually fit text into some arbitrary text-block max char per line.